aws-shared-1/main.tf

```diff
@@ -97,6 +97,28 @@ module "aws_az_1e" {
   workers_org_subnet_cidr = "${var.workers_org_subnet_1e_cidr}"
 }
 
+resource "aws_route53_record" "workers_org_nat" {
+  zone_id = "${var.travisci_net_external_zone_id}"
+  name = "workers-nat-org-${var.env}-${var.index}.aws-us-east-1.travisci.net"
+  type = "A"
+  ttl = "300"
+  records = [
```

Comment:
only reason I can think of that we'd be concerned about having that amount of granularity is if something was going wrong in one AZ but not in another -- but if we were to want to get into a specific bastion host, it'd be a bit of a treasure hunt to find out which IP is which host.

also, does route53 do reverse dns as well? b/c if that's the case then we could at least depend on reverse dns to be able to get the AZ out of the respective IPs.